### ISSUE TITLE
Punctuation and digits are always adding to the beginning and/or end of a password

### ISSUE THREAD

**Issue by charl** [michaelbironneau/garbler#3]
Distributing punctuation and digits throughout the passwords will most likely provide a more random (stronger?) password, sacrificing _memorability_.

Would be nice if this was a switch that was off by default.

**Comment by charl** [comment]
Apg does this by joining the memorable bits with punctuation and/or digits.

**Comment by michaelbironneau** [comment]
You're right, the way Apg does it makes good sense. Will have a look at implementing this.

Maybe the way forward is an option that specifies how memorable the password should be - there are probably a few other optional modifications that could strengthen the password at the expense of memorability.
